tests/testthat/test-packrat.R

```diff
@@ -85,15 +85,17 @@ withTestContext({
     # Snapshot the new state and make sure we picked up both toast and its
     # dependency, bread
     pkgs <- pkgNames(lockInfo(projRoot))
-    expect_false("bread" %in% pkgs)
-    expect_false("toast" %in% pkgs)
-    snapshot(projRoot)
+    msg <- paste0("pkgs: ", pkgs, collapse = ", ")
+    expect_false("bread" %in% pkgs, msg)
+    expect_false("toast" %in% pkgs, msg)
+    snapshot(projRoot, ignore.stale = TRUE)
```

Comment:
These tests have forever been a struggle to have pass locally. This may not be the final fix, but lets them pass again. @kevinushey - do you have any ideas about why these tests were failing when run with `devtools::check(remote = TRUE, manual = TRUE)`?

Before the change, we were seeing:

```
The following packages are stale:
            _        
    packrat   0.9.1-1

These packages must be updated by calling packrat::restore() before
snapshotting. If you are sure you want the installed versions of these
packages to be snapshotted, call packrat::snapshot() again with
ignore.stale=TRUE.
--
Snapshot operation was cancelled, no changes were made.
```